doc/users/next_whats_new/new_arrow_param.rst

```diff
@@ -0,0 +1,5 @@
+New parameter ``ArrowStyle._Curve.arrow``
+-----------------------------------------
+
+The ``ArrowStyle._Curve.arrow`` parameter accepts arrows like '<-', '-['
```

Comment:
Thanks for your persistence!  OTOH, I'm not seeing a test for this new functionality.  Did I simply miss it?

Reply:
`test_arrow_styles` plots all styles (cf, the changed test image).

Reply:
Like @QuLogic said, the `test_arrow_styles` will create an image with all the possible arrows and then compare it with the `lib/matplotlib/tests/baseline_images/test_arrow_patches/arrow_styles.png`. For this reason this and the `lib/matplotlib/tests/baseline_images/test_path/arrow_contains_point.png` needed to be changed.

Reply:
Sorry to be dense, but surely our public interface to this is not `ArrowStyle._Curve`?  The what's new should say what is new from the user's point of view.

Reply:
Looking this way, would it be better to document it as changes in the `ArrowStyle.Curve`, `ArrowStyle.CurveA`... , instead of `ArrowStyle._Curve`?

Reply:
Yes, if that is the proper public interface (I'm not following this too closely, but we shouldn't mention private classes in the public docs, unless absolutely necessary).

Reply:
Yes, `ArrowStyle`, but also the `arrowstyle` argument to `ConnectionPatch` accepts the new combinations, which I think is the even more user-facing side of things.